a/amazon-metadata.yml

```diff
@@ -0,0 +1,19 @@
+amazon-metadata:
+  image: rancher/os-amazon-metadata:v0.9.2${SUFFIX}
```

Comment:
BTW, [build-images](/rancher/os-services/scripts/build-images) script updates version and rebuilds image automatically. If you're going to use separate images, you'll need to fork the repos with them to rancher organization (just to make it all be in one place). Just like for [crontab.yml](/rancher/os-services/c/crontab.yml)
My repos with this images code are:
https://github.com/efim-a-efim/rancheros-ec2-metadata
https://github.com/efim-a-efim/docker-volume-netshare-container